diff --git a/libavfilter/Makefile b/libavfilter/Makefile
index e06acdf..7edf7c5 100644
--- a/libavfilter/Makefile
+++ b/libavfilter/Makefile
@@ -541,0 +542,2 @@ OBJS-$(CONFIG_ZSCALE_FILTER)                 += vf_zscale.o
+OBJS-$(CONFIG_GLTRANSITION_FILTER)           += vf_gltransition.o
+OBJS-$(CONFIG_SHADERTOY_FILTER)              += vf_shadertoy.o
diff --git a/libavfilter/allfilters.c b/libavfilter/allfilters.c
index 2ad523f..278dd1f 100644
--- a/libavfilter/allfilters.c
+++ b/libavfilter/allfilters.c
@@ -512,0 +513,2 @@ extern const AVFilter ff_vf_zscale;
+extern const AVFilter ff_vf_gltransition;
+extern const AVFilter ff_vf_shadertoy;
